hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/v2/app/job/impl/TaskAttemptImpl.java

```diff
@@ -1970,6 +1976,10 @@ private void logAttemptFinishedEvent(TaskAttemptStateInternal state) {
     @SuppressWarnings("unchecked")
     @Override
     public void transition(TaskAttemptImpl taskAttempt, TaskAttemptEvent event) {
+      if (taskAttempt.getInternalState() ==
+          TaskAttemptStateInternal.SUCCESS_FINISHING_CONTAINER){
+      sendContainerCleanup(taskAttempt, event);
```

Comment:
Indentation looks off here.
```suggestion
        sendContainerCleanup(taskAttempt, event);
```